docs/modules/operation/pages/database-reports/templates/calendar.adoc

```diff
@@ -0,0 +1,35 @@
+
+[[calendar-report]]
+= Default Calendar Report
+
+Standard OpenNMS report in calendar format.
+
+This report is generated directly from the {page-component-title} Java code and has no optional parameters for selecting a subset of the network devices.
+The report contains the same information as the "Default Classic Report" with the exception that the information in the second and third tables presents the data in the format of a calendar month.
+
+The period is the current month to the last full day before the day the report was generated (in other words, yesterday).
+All the data is presented in tables.
+
+The first table contains a table of the monthly availability percentage for all services {page-component-title} monitored for the past year.
+It also lists the number of nodes that had outages, the number of interfaces, and the number of services.
+// QUESTION: Not sure if this last set of information show the maximum or current number of interfaces and services.
```

Comment:
@mmahacek did something go wrong? I thought that Andrew had removed all the questions.